### DISCUSSION TITLE
Not starting

### DISCUSSION THREAD

**Discussion by ghost** [Vedza/NitroSniperGo#302] [Q&A]
Ah yes hi, For some reason the sniper doesn't want to start, (https://axmyois.an-idiot.xyz/koyioq0669a) I get this and then nothing happens..

**Comment by Vedza** [comment]
Maybe you have the same issue as this one https://github.com/Vedza/NitroSniperGo/issues/295

**Reply by ghost** [reply]
okay, ima try that!
